glide.go

```diff
@@ -34,7 +34,7 @@ import (
 	"os"
 )
 
-var version = "0.13.0-dev"
+const version = "0.13.0-dev"
```

Comment:
By changing from a `var` to `const` the `ldflags` for `-X main.version=${VERSION}` no longer works in the `make build`. This is why it was a `var` in the first place.

I think this needs to be switched back.